articles/terraform/create-linux-virtual-machine-with-infrastructure.md

```diff
@@ -248,7 +248,7 @@ To use SSH to connect to the virtual machine, do the following steps:
 1. Run [terraform output](https://www.terraform.io/cli/commands/output) to get the SSH private key and save it to a file.
 
     ```console
-    terraform output -raw tls_private_key > id_rsa
+    terraform output -raw tls_private_key > ~/id_rsa
```

Comment:
If we put id_rsa inside cloudsorage path, we can not change the permission to 600. But, ~/ works.